app/templates/components/create-options-dates.hbs

```diff
@@ -3,10 +3,22 @@
   property="options"
   data-test-form-element-for="days"
 }}
-  <InlineDatepicker
-    @center={{calendarCenter}}
-    @selectedDays={{selectedDays}}
-    @onCenterChange={{action (mut calendarCenter) value="moment"}}
-    @onSelect={{action "daysSelected"}}
-  />
+  <div class="row">
+    <div class="col-12 col-md-6">
+      <InlineDatepicker
+        @center={{calendarCenter}}
+        @selectedDays={{selectedDays}}
+        @onCenterChange={{action (mut calendarCenter) value="moment"}}
+        @onSelect={{action "daysSelected"}}
+      />
+    </div>
+    <div class="col-6 cr-hide-on-mobile">
```

Comment:
I was wondering why there is `.col-6` and not `.col-md-6` here. If I got it right, `.cr-hide-on-mobile` hides the row if screen size is bigger than `md`, isn't it? I think it would be easier to reason about if having `col-md-6` even if it's not necessary.

Why are you using custom `.cr-hide-on-mobile` and not [Bootstrap 4 utilty classes to hide these elements for certain breakpoints](https://getbootstrap.com/docs/4.0/utilities/display/#hiding-elements)? If I didn't missed something `.d-none .d-md-block` should do the same.

Reply:
> I was wondering why there is .col-6 and not .col-md-6 here. If I got it right, .cr-hide-on-mobile hides the row if screen size is bigger than md, isn't it? I think it would be easier to reason about if having col-md-6 even if it's not necessary.

Okay. I get that.

> Why are you using custom .cr-hide-on-mobile and not Bootstrap 4 utilty classes to hide these elements for certain breakpoints? If I didn't missed something .d-none .d-md-block should do the same.

I wanted to keep the CSS bundle small and including all the utilities (or even just all the display utilities) seems like a hefty price to pay for this small piece of functionality. That's why I didn't use it.

Reply:
Fixed the column class issue.

Reply:
> I wanted to keep the CSS bundle small and including all the utilities (or even just all the display utilities) seems like a hefty price to pay for this small piece of functionality. That's why I didn't use it.

That makes sense. I'm wondering if we could cherry pick the utility classes in some way. Would increase readability for everyone familiar with Bootstrap while not blowing up asset size.

Reply:
It would be possible to define a custom utilities config to generate only certain classes. (https://github.com/twbs/bootstrap/blob/master/scss/_utilities.scss) However not sure if this is a good idea because although that would be a more standards-compatible way it actually adds a whole new level of indirection. I think this might be a bit too much for just one class.